bin/desi_proc

```diff
@@ -22,18 +22,23 @@ time srun -n 20 -N 1 -C haswell -t 15:00 --qos realtime desi_proc --mpi -n 20191
 """
 
 import time
-from collections import OrderedDict
-
-progress = OrderedDict(start=time.asctime())
+start_imports = time.time()
 
+from collections import OrderedDict
```

Comment:
Some of these moved imports aren't picked up as the same. Is there a motivation behind the moves or was this a result of adding and removing code during development?

It isn't a blocking factor but it does increase the differences with master that will impact the upcoming rebase of the desi_proc_mod branch.

Reply:
It was mostly a side effect of rearranging imports to be able to time them properly (`import time` and grab `time.time()` ASAP; wait on MPI until after argparsing; wait on `freeze_iers` until we've got the timers going...)  For now I reverted the order to close to the original with only the minimally necessary rearrangement.

At the same time, our import order has gotten a bit messy compared to our (perhaps undocumented?) typical order:
  * python standard libraries (os, sys, glob, ...)
  * 3rd party libraries (numpy, scipy, astropy, ...)
  * other desi packages (desiutil, desitarget, ...)
  * other modules from within this package (desispec.io, desispec.sky, ...)

We can standardize that in a later update.

Reply:
Thank you for reverting. We can refactor right after rebase so you can capture the timing appropriately.